React/src/App.js

```diff
@@ -0,0 +1,88 @@
+import React , {Component} from 'react';
+import logo from './logo.svg';
+import './App.css';
+import web3 from './web3';
+import lottery from './lottery';
+
```

Comment:
Add Components according to the dashboard.

Reply:
On it